v2/types.go

```diff
@@ -375,6 +408,12 @@ type UnbindRequest struct {
 	InstanceID string `json:"instance_id"`
 	// BindingID is the ID of the binding to delete.
 	BindingID string `json:"binding_id"`
+	// AcceptsIncomplete indicates whether the client can accept asynchronous
```

Comment:
Can you make a not about this and the equivalent field on BindRequest being alpha?